modules/vaos/spec/controllers/v2/eps_appointments_controller_spec.rb

```diff
@@ -0,0 +1,182 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe VAOS::V2::EpsAppointmentsController, type: :request do
```

Comment:
Note: There was no EPS controller test, so here I created a new one to cover as much as possible.